config/keyboards/kanata.kbd

```diff
@@ -1,8 +1,8 @@
-;; (defcfg) ;; Enable this line except use via nix
+(defcfg concurrent-tap-hold yes)
```

Comment:
Since enabling `concurrent-tap-hold yes`, it looks buggy :thinking: `git push origin main` will always be `git push origin mian`

https://github.com/jtroo/kanata/blob/0b25d28fd2e06d82e9e2060f88d57cd3f005c981/docs/config.adoc?plain=1#L3301-L3309

Reply:
Added `f1 a c` in defsrc, `a` might be unstable

Reply:
Adjusted in https://github.com/kachick/dotfiles/commit/352bf4fcb8bdd559944ffba4b9a256c48e35678b